src/Components/WebAssembly/Compression/src/Program.cs

```diff
@@ -36,7 +36,7 @@ static async Task<int> Main(string[] args)
                 var targetCompressionPath = file.Target;
 
                 if (!File.Exists(inputSource) ||
-                    (File.Exists(targetCompressionPath) && File.GetLastWriteTime(inputSource) > File.GetLastWriteTime(targetCompressionPath)))
+                    (File.Exists(targetCompressionPath) && File.GetLastWriteTime(inputSource) < File.GetLastWriteTime(targetCompressionPath)))
```

Comment:
Same suggestion as the other code path. You could consider rewriting this to remove an additional `File.Exists` per invocation.